docs/BR.md

```diff
@@ -428,6 +428,8 @@ The script outputs:
 
 **Root Certificate**: The self-signed Certificate issued by the Root CA to identify itself and to facilitate verification of Certificates issued to its Subordinate CAs.
 
+**Short-lived Subscriber Certificate**: A server Certificate with a Validity Period less than ten (10) days (864,000 seconds). 
```

Comment:
Can the Validity Period be defined as a shorter period, e.g. five days, 432,000 seconds?